colorable_others.go

```diff
@@ -1,4 +1,4 @@
-// +build !windows
+// +build !windows OR appengine
```

Comment:
Fixed to `// +build !windows appengine`. (I misunderstood the syntax for build constraints)
I want go-colorable use `colorable_others.go` when "not on windows (e.g. Linux)" OR "on windows with appengine".